taxonomies/ingredients.txt

```diff
@@ -22955,6 +22959,10 @@ nl:volkorengerst
 pl:jeçzmien z pełnego przemiału
 pt:cevada integral
 
+<en:barley
+en:chopped barley
```

Comment:
we will be removing entries like "chopped XYZ", as the ingredients-processing.txt taxonomy will take care of it. so we should not add new ones. instead we can make sure the ingredient-processing taxonomy has the correct Spanish entry.

Reply:
ok, I'm passing all through the ingredients analysis so adding ones that don't pass